### ISSUE TITLE
[Keyboard Navigation  - MyAccess - Filters] Control type is not define for 'checkbox' under 'Filter' blade.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#10339]
**User Experience:**
When standard controls from accessible technologies are not used and if the user interface elements are not used according to specification, Assistive Technologies (AT) will not be able to gather information about, activate(or set) and keep up to date on the status of user interface controls in the content.
If custom controls are created, however, or interface elements are programmed (in code or script) to have a different role and/or function than usual, and controls does not provide important information to assistive technologies, it would be very difficult for AT dependent users.

**Test Environment:**
Version 1809 (OS Build 18314.1000)
Platform: Edge, Chrome
Screen reader: Narrator, JAWS

**Repro steps:**
1. Open URL 'https://myaccess.microsoft.com/@M365x202152.onmicrosoft.com#/access-packages' and enter valid credentials.
2. Navigate to 'Access Package' list item and select it.​
3. 'Access Package' screen will open.​
4. Navigate to 'Active' tab items and select it.​
5. 'Active' packages screen will open and select any active package like 'test' from table.​
6. Navigate to 'Filter' button and select it.​
7. 'Filters' screen will open.​
8. Navigate to various controls such as radio buttons on the screen.​
9. Verify that  Control type is define or not for 'checkbox' control.

**Actual Result:**
Control type is not define for 'checkbox' control under 'Filter' blade.

**Expected Result:**
Control type (Name, role value) should be define for 'checkbox' under 'Filter' blade.

**MAS Reference:** 
MAS 4.1.2 - Name, Role, Value

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

![MAS 4 1 2_Checkbox](https://user-images.githubusercontent.com/30456910/64169149-b6308880-ce6a-11e9-81a9-bcf171e9328d.png)

**Comment by KevinTCoughlin** [comment]
Similar issue to #10338 in that the control houses a font character as its text which the accessibility tool flags as invalid. I'll need to dig in to see what the appropriate fix is either in Fabric or user override.

**Comment by dzearing** [comment]
This is not a Fabric error. This is the user implementing their own checkbox using the presentational `Check` component. The user should either use `Checkbox`, which has the correct labeling, or wrap the `Check` in an interactive component that reimplements the Checkbox accessibility with the correct roles.
